### DISCUSSION TITLE
Feature request: and/or Rule triggers

### DISCUSSION THREAD

**Discussion by supercrisz** [firefly-iii/firefly-iii#9329] [Firefly III] [answered]
I don't know if this is the right place for a feature request. Now the strict mode is not flexible: you can choose all or nothing to be strict. But it would be great to have an And/Or rule creating flow. For example: i want a flow like this

**OR** 
     account name = abc **AND** amount > 90 
**OR**
     account name = xyz
     
Now, I need to create 2 seperate rules to accomplish this so also have to manage two rules if something needs to change on labeling. To be able to create a rule via And/Or flow, it is less time consuming on management of rules. Is it an idea to have it on future releases?

**Comment by supercrisz** [comment]
I added this to Q&A category, maybe then more appearence on this topic :)

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/explanation/more-information/what-its-not/#more-complicated-rules-like-boolean-logic-copypasting-values-etc
